tests/test_arbitrary_package_attack.py

```diff
@@ -102,8 +92,10 @@ def setUp(self):
     # any modifications made to repository files are restricted to the copies.
     # The 'repository_data' directory is expected to exist in 'tuf/tests/'.
     original_repository_files = os.path.join(os.getcwd(), 'repository_data')
+    # Create a temporary directory to store the repository, metadata,
+    # and target files.
```

Comment:
I don't think we need to include this comment here or throughout the tests you've edited.